src/js/components/intro.js

```diff
@@ -0,0 +1,33 @@
+import { newText, newMultimedia, newContainer } from './helpers';
+
+/**
+ * @param {Array [objectImages]} imagesArrays es un array de imagenes de la data
+ * Toma el array de 'imagesArrays' le coloca un map que nos crean una imagen por elemento
+ * del array y nos coloca una clase y un modificador que cambia según el número de elementos llamados y nos retorna
+ * un array de nodos de imagénes.
+ */
+const nodeImages = (imagesArrays) => {
+  const imagesList = imagesArrays.map((image, index) => {
+    const imageIntro = newMultimedia('img', image, ['Intro__image', `Intro__image--${index}`]);
```

Comment:
the class name, always starts with lower case.
I will recommend changing this in all the class names